client/src/components/PopularShows.js

```diff
@@ -33,24 +37,24 @@ function PopularShows({ tab }) {
     toggleModal();
   }
 
-  if ((trendingStatus || newestStatus) === 'loading') {
-    return <span>Loading...</span>;
-  }
-
-  if ((trendingStatus || newestStatus) === 'error') {
-    return <span>Error</span>;
-  }
-
   return (
     <>
       <ShowDetailViewModal
         isShowing={isShowing}
         toggleModal={toggleModal}
         showDetails={selectedItem}
       />
+      {tab === 'newest' &&
+        newestShowsError &&
+        toast.error(newestShowsError.message, {
+          closeOnClick: true,
+          closeButton: true,
+          autoClose: '5000',
+        })}
```

Comment:
You will show this toast anytime this component is rendered. Better move your toasts to a `useEffect`.